### DISCUSSION TITLE
How to auto remove collection item after one week expiration

### DISCUSSION THREAD

**Discussion by creotip** [strapi/strapi#8210] [Legacy Topics]
Searched for a solution everywhere,  but without success.
Strapi corn jobs down not work with pm2, so there is any solution for that?

**Comment by derrickmehaffy** [comment]
Hello,

I see you are wanting to ask a question, we are currently planning to decomission our GitHub Discussions in early 2021 and we don't want to lose the great questions asked by our community.

Questions should be directed to [our forum](https://forum.strapi.io/). Please see the following contributing guidelines for asking a question [here](https://github.com/strapi/strapi/blob/master/CONTRIBUTING.md#reporting-an-issue).
